examples/pylab_examples/quiver_demo.py

```diff
@@ -37,7 +23,6 @@
 plt.axis([-1, 7, -1, 7])
 plt.title('scales with plot width, not view')
```

Comment:
can these be at the top of the plot code? I tend to scan for the title code and then expect the rest of the code underneath it to make the new figure.